man/systemd.netdev.xml

```diff
@@ -1848,6 +1848,28 @@
     Linux Ethernet Bonding Driver HOWTO</ulink></para>
   </refsect1>
 
+  <refsect1>
+    <title>[Xfrm] Section Options</title>
+
+    <para>The <literal>[Xfrm]</literal> section accepts the following
+    keys:</para>
+
+    <variablelist class='network-directives'>
+      <varlistentry>
+        <term><varname>Device=</varname></term>
+        <listitem>
+          <para>Sets the underlaying interface. Can be "lo".</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>InterfaceId=</varname></term>
+        <listitem>
+          <para>Sets the ID of the xfrm device.</para>
```

Comment:
that's not a useful help string. it says pretty much as much what I could guess from the setting name. Please explain here what this "id" is supposed to mean